interpreter/php/php.go

```diff
@@ -41,7 +41,7 @@ var (
 	evalCodeFunctionName = libpf.Intern("<eval'd code>")
 
 	// regex for the interpreter executable
-	phpRegex     = regexp.MustCompile(".*/php(-cgi|-fpm)?[0-9.]*$|^php(-cgi|-fpm)?[0-9.]*$")
+	phpRegex     = regexp.MustCompile(".*/php(-cgi|-fpm)?[0-9.]*$|^php(-cgi|-fpm)?[0-9.]*$|.*libphp.*\\.so")
```

Comment:
```suggestion
	phpRegex     = regexp.MustCompile(`.*/php(-cgi|-fpm)?[0-9.]*$|^php(-cgi|-fpm)?[0-9.]*$|.*libphp.*\.so`)
```

Reply:
Note that I split the regexp string literal into two lines to make linter happy.